ibc-tests/tests/validator_staking.rs

```diff
@@ -113,14 +104,6 @@ fn test_validator_staking() {
     validator_staking_contract
         .execute(&unstake_msg, None)
         .unwrap();
-    let contract_balance = daemon
```

Comment:
why were these checks removed?

Reply:
In e2e test, the exact balance can not be estimated.
1. In case `WithdrawAddrEnabled` parameter is deactivated(set to `false`), previously claimed funds will be on the contract.
2. The exact reward amount can not be calculated due to timestamp difference(query time  and actual transaction time)
3. Fee problem was also tricky
The main purpose of the e2e test is to ensure that no funds are locked on the validator staking contract and thus removed the contract balance checks before `withdraw_funds`